blacksauce/src/styles/grid/_footerGrid.scss

```diff
@@ -0,0 +1,65 @@
+//general grid for the social media
+.grid--socialMedia {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+}
+
+//icons for social media
+.facebookIcon {
```

Comment:
I don't think the column declarations are necessary for any of these social media icons; this should be where they layout by default.